src/styles/blocks/header.scss

```diff
@@ -0,0 +1,50 @@
+.container {
+  margin: 0 auto;
+  padding: 24px 18px 0;
```

Comment:
It's a good practice to use variables for common padding and margin values. Consider creating variables for the padding values in '_variables.scss' and use them here.